src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -16,7 +16,18 @@ namespace System.Net.Http.Tests
     public class HttpClientHandlerTest
     {
         readonly ITestOutputHelper _output;
+        private const string Username = "testuser";
+        private const string Password = "password";
+        private const string DecompressedContentPart = "Accept-Encoding";
+        private readonly NetworkCredential _credential = new NetworkCredential(Username, Password);
 
+        private static bool JsonMessageContainsKeyValue(string message, string key, string value)
+        {
+            // Poor Man's json parsing.
```

Comment:
it's probably because the current test server is the httpbin.org server which sends responses back in JSON format.

That's not going to be the test server long term. But it's possible that the test server might use JSON responses (although our current ToF test server don't).